ui/src/app/lib/atlasmap-data-mapper/models/document.definition.model.ts

```diff
@@ -56,65 +56,28 @@ export class NamespaceModel {
     }
 }
 
-export enum DocumentTypes { JAVA, XML, JSON, CSV, CONSTANT, PROPERTY}
-
-export class DocumentType {
-    public type: DocumentTypes = DocumentTypes.JAVA;
-
-    public isJava(): boolean {
-        return this.type == DocumentTypes.JAVA;
-    }
-
-    public isXML(): boolean {
-        return this.type == DocumentTypes.XML;
-    }
-
-    public isJSON(): boolean {
-        return this.type == DocumentTypes.JSON;
-    }
-
-    public isCSV(): boolean {
-        return this.type == DocumentTypes.CSV;
-    }
-
-    public isConstant(): boolean {
-        return this.type == DocumentTypes.CONSTANT;
-    }
-
-    public isProperty(): boolean {
-        return this.type == DocumentTypes.PROPERTY;
-    }
-
-    public isPropertyOrConstant(): boolean {
-        return this.isProperty() || this.isConstant();
-    }
-}
+export class DocumentDefinition {
+    public id: string;
+    public _type: DocumentType;
+    public shortName: string;
+    public fullName: string;
+    public uri: string;
+    public inspectionType: InspectionType;
+    public inspectionSource: string;
+    public inspectionResult: string;
+    public isSource: boolean;
+    public isPropertyOrConstant: boolean;
```

Comment:
Just a note for the future: Class members are `public` by default, so the standard convention is NOT to annotate public fields with the `public` accessor. This is just a remark, so please leave this file as it is since adding the `public` accessor is not wrong either (although redundant).

Reply:
Thanks, let's cleanup before I forget ;-)